front/components/assistant/conversation/ConversationMessage.tsx

```diff
@@ -29,7 +34,17 @@ export function ConversationMessage({
         </div>
       </div>
       <div className="flex flex-col items-start gap-2 sm:flex-row">
-        {buttons}
+        {buttons &&
+          buttons.map((button) => (
+            <div key={uuid4()}>
```

Comment:
The goal of react keys is to prevent re-render so you definitely don't want uuid generated stuff which would re-render at all time

Reply:
My bad :bow: